website/docs/user-group/user/user_basic_operations.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Manage users
+---
+
+The following topics are for the basic management of users: how to create, modify, delete or deactivate users, and using a recovery email.
+
+### Create a user
+
+> If you want to automate user creation, you can do that either by [invitations](./invitations.md), [user write stage](../../flow/stages/user_write), or [using the API](/developer-docs/api/browser).
+
```

Comment:
> If you can automate creating new users by:
>
> - [Invitations](./invitations.md): Invite your users to create their own accounts
> - [Using the API](./developer-docs/api/browser): Create user accounts through automated scripting

Looking at the contents of `/stages/user_write` in the API (I couldn’t find `/flow/stages/user_write` in the API, and the URL `http://localhost:9000/if/admin/#/flow/stages/user_write` in the WebUI gives me a 404), this looks much more like “how I define the user_write stage for a new user flow,” and not “how I define a new user and use the Flow to put them into the system.”

```Javascript
{
    name: string
    flow_set: [{
        name: string
        slug: string
        title: string
        designation: enum
        policy_engine_mode: enum
        compatibility_mode: boolean
        layout: enum
        denied_action: enum
    }]
    user_creation_mode: enum
    create_users_as_inactive: boolean
    user_path_template: string
}
```

Reply:
In a future PR I will attempt to clarify the use of the `user_write` stage. I just want to get these basic CRUD procedurals out now.